doc/api/child_process.md

```diff
@@ -147,7 +147,9 @@ added: v0.1.90
 * Returns: {ChildProcess}
 
 Spawns a shell then executes the `command` within that shell, buffering any
-generated output.
+generated output. When specifying paths, spaces need to be dealt with 
```

Comment:
This isn't path specific, what about `$`? Or args that are not paths? Can you change to make clear that the command line is being passed to the shell, and will be processed as a shell command, so beware of anything significant: white space, shell meta chars, etc.